win32.c

```diff
@@ -21,6 +21,26 @@ typedef UINT32 *PQOS_FLOWID;
 #endif
 
 static enet_uint32 timeBase = 0;
+
+#if defined(WINAPI_FAMILY) && WINAPI_FAMILY == WINAPI_FAMILY_APP
```

Comment:
I'd prefer if you just used `defined(WINAPI_FAMILY) && WINAPI_FAMILY == WINAPI_FAMILY_APP` everywhere instead of defining `ON_UWP` and using that.

Reply:
Actually, just invert this and call it `HAS_QWAVE`.